cpu/esp32/startup.c

```diff
@@ -188,12 +188,67 @@ NORETURN void IRAM call_start_cpu0 (void)
 extern uint32_t esp_clk_slowclk_cal_get(void);
 extern void IRAM_ATTR rtc_select_slow_clk(rtc_slow_freq_t slow_clk);
 
+
+// <<<< BEGIN PROVISIONAL CODE
+// Note: Copied verbatim from IDF's bootloader_clock.c except where noted.
+// I have no idea whether setting the CPU frequeny to 80MHz is needed, but it
+// looks like rtc_clk_init() needs _some_ value to set it to. --snej
+static void bootloader_clock_configure(void)
```

Comment:
Please move this function either to file ```cpu/esp32/vendor/esp-idf/esp_funcs.c``` which is a collection of single functions that were extracted from different ESP-IDF files or create a new subdirectory ```cpu/esp32/vendor/esp-idf/bootloader_support``` and place the original file with your modifications there for the following reasons:

- The function is extracted from ESP-IDF as it is.
- since all files in ```cpu/esp32/vendor/``` are vendor files, it is not necessary to fulfill RIOT's coding style where ```//``` comments are not allowed. Furthermore, it becomes clear that the code was extracted from ESP-IDF.
- I prefere to keep ```startup.c``` small and not to overload it with supporting function code.

Reply:
Will do.